tests/distributed/test_mp_dataloader.py

```diff
@@ -341,7 +339,6 @@ def test_dist_dataloader(
             test_dir,
             num_hops=num_hops,
             part_method="metis",
-            reshuffle=reshuffle,
```

Comment:
seems forgot to remove `@pytest.mark.parametrize("reshuffle", [True, False])` in the test func

Reply:
Nice catch! Removed for fix